app/src/main/java/eu/kanade/domain/manga/model/Manga.kt

```diff
@@ -99,6 +99,28 @@ data class Manga(
         it.initialized = initialized
     }
 
+    fun copyFrom(other: SManga): Manga {
+        val author = other.author ?: author
+        val artist = other.artist ?: artist
+        val description = other.description ?: description
+        val genres =  if (other.genre != null) {
```

Comment:
```suggestion
        val genres = if (other.genre != null) {
```
I typed an extra space here, sorry.